src/utils/events.ts

```diff
@@ -20,20 +19,16 @@ type EventsListData = {
 };
 
 export const fetchEvents = async (limit: number, offset: number): Promise<EventsListData> => {
-  const currentDate = new Date();
-  currentDate.setDate(currentDate.getDate() - 1);
-  const afterDate = currentDate.toISOString().split('T')[0]; // YYYY-MM-DD
-
-  const queryFrom = `after=${afterDate}`;
+  const eventsApiUrl = 'https://api.lu.ma';
+  const queryFrom = `period=future`;
   const queryLimit = `pagination_limit=${limit ?? 10}`;
   const queryOffset = offset ? `pagination_offset=${offset}` : '';
   const queryParams = [queryFrom, queryLimit, queryOffset].filter(Boolean).join('&');
 
-  const res = await fetch(`${eventsApiUrl}/calendar/list-events?${queryParams}`, {
+  const res = await fetch(`${eventsApiUrl}/calendar/get-items?calendar_api_id=cal-Nrz4EsmLDjXvjPp&${queryParams}`, {
```

Comment:
should [these configs](https://github.com/near/nearorg/blob/develop/src/utils/config.ts#L1) be used here?  or should config.ts be deleted?

Reply:
Yes, you're right. From this moment that file should be deleted. Will do this in next commit